libs/coin-modules/coin-sui/src/network/sdk.test.ts

```diff
@@ -886,20 +886,46 @@ describe("Staking Operations", () => {
       expect(operation.asset).toEqual({ type: "native" });
       expect(operation.tx.block.hash).toBeUndefined();
     });
-  });
 
-  describe("Operation Extra Information", () => {
-    test("getOperationExtra should be a function", () => {
-      expect(typeof sdk.getOperationExtra).toBe("function");
+    test("transactionToOp should include stakedAmount in details for DELEGATE operations", () => {
+      const address = "0x65449f57946938c84c512732f1d69405d1fce417d9c9894696ddf4522f479e24";
+
+      const operation = sdk.alpacaTransactionToOp(address, mockStakingTx(address, "-1001050000"));
+
+      expect(operation).toMatchObject({
+        type: "DELEGATE",
+        details: {
+          stakedAmount: 1000000000n,
+        },
+      });
     });
 
-    test("getOperationExtra should return a Promise", () => {
-      const result = sdk.getOperationExtra("test_digest");
-      expect(result).toBeInstanceOf(Promise);
+    test("transactionToOp should include stakedAmount in details for UNDELEGATE operations", () => {
+      const address = "0x65449f57946938c84c512732f1d69405d1fce417d9c9894696ddf4522f479e24";
+
+      const operation = sdk.alpacaTransactionToOp(address, mockUnstakingTx(address, "998950000"));
+
+      expect(operation).toMatchObject({
+        type: "UNDELEGATE",
+        details: {
+          stakedAmount: 1000000000n,
+        },
+      });
     });
   });
 });
 
+describe("Operation Extra Information", () => {
+  test("getOperationExtra should be a function", () => {
+    expect(typeof sdk.getOperationExtra).toBe("function");
```

Comment:
```suggestion
expect(sdk.getOperationExtra).toBeInstanceOf(Function);
```
`typeof` is avoidable